.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "pip" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+      time: "08:00"
+  - package-ecosystem: "npm" # See documentation for possible values
```

Comment:
Why do we need npm here? 

I see a package.json that refers to a sass build process but I don't see any of the associated directories. Is the re a pipeline that runs for this repo to produce assets?